apps/prairielearn/src/lib/externalGraderResults.ts

```diff
@@ -146,19 +147,33 @@ async function processMessage(data: {
     ...data,
   });
   if (data.event === 'job_received') {
-    await sqldb.queryOneRowAsync(sql.update_grading_received_time, {
+    await sqldb.queryAsync(sql.update_grading_received_time, {
       grading_job_id: jobId,
       received_time: data.data.receivedTime,
     });
     await gradingJobStatusUpdated(jobId);
     return;
   } else if (data.event === 'grading_result') {
     // Figure out where we can fetch results from.
-    const jobDetails = await sqldb.queryOneRowAsync(sql.get_job_details, {
-      grading_job_id: jobId,
-    });
-    const s3Bucket = jobDetails.rows[0].s3_bucket;
-    const s3RootKey = jobDetails.rows[0].s3_root_key;
+    const jobDetails = await sqldb.queryRow(
+      sql.get_job_details,
+      {
+        grading_job_id: jobId,
+      },
```

Comment:
See #12499.
```suggestion
      { grading_job_id: jobId },
```

Reply:
> has an automated way of formatting and fixing this

The automated way was "some weird little bash one-off" 😅

Agreed that something like #11410 is too opinionated/invasive. In the link Jonatan shared above, Prettier states that "We haven’t been able to find a good rule for all those cases". I largely agree with this: IMO what's best is subjective.

If it bothers you to receive these nits, we can take the same approach we adopted with documentation nits and encourage the reviewer to just silently push the desired changes to the branch.

Reply:
> If it bothers you to receive these nits, we can take the same approach we adopted with documentation nits and encourage the reviewer to just silently push the desired changes to the branch.

I would prefer to not receive formatting nits, and would prefer @jonatanschroeder /others so push to the branch. If it is so opinionated that a rule can't be enabled to enforce the desired style, I don't think it's worth the energy going through a review cycle for. If there _is_ a rule, I am happy to add it!

Reply:
I pushed the reformatting to this branch. We can race it against https://github.com/PrairieLearn/PrairieLearn/pull/12517, which does the same thing for all queries on master (there were a few I missed the other day).